src/components/BrowseResultsList/utils.js

```diff
@@ -103,22 +103,47 @@ export const getFullCallNumber = (row) => {
   return fullCallNumber;
 };
 
+const getCallNumberQuery = (qindex, data, row) => {
```

Comment:
A function header that describes the arguments and return values sure would be nice.